dao/src/main/java/com/epam/esm/dao/entity/GiftCertificate.java

```diff
@@ -1,124 +1,50 @@
 package com.epam.esm.dao.entity;
 
+import lombok.*;
```

Comment:
please do not use wildcards in the `import` statement, It's a bad habit.
Fix it in the whole project